thundra/constants.py

```diff
@@ -10,9 +10,10 @@
 THUNDRA_DISABLE_TRACE = 'thundra_disable_trace'
 THUNDRA_DISABLE_METRIC = 'thundra_disable_metric'
 THUNDRA_LAMBDA_PUBLISH_CLOUDWATCH_ENABLE = 'thundra_lambda_publish_cloudwatch_enable'
-THUNDRA_LAMBDA_AUDIT_REQUEST_SKIP = 'thundra_lambda_audit_request_skip'
-THUNDRA_LAMBDA_AUDIT_RESPONSE_SKIP = 'thundra_lambda_audit_response_skip'
+THUNDRA_LAMBDA_TRACE_REQUEST_SKIP = 'thundra_lambda_trace_request_skip'
+THUNDRA_LAMBDA_TRACE_RESPONSE_SKIP = 'thundra_lambda_trace_response_skip'
 THUNDRA_APPLICATION_PROFILE = 'thundra_applicationProfile'
 AWS_LAMBDA_LOG_STREAM_NAME = 'AWS_LAMBDA_LOG_STREAM_NAME'
 AWS_LAMBDA_FUNCTION_VERSION = 'AWS_LAMBDA_FUNCTION_VERSION'
-AWS_REGION = 'AWS_REGION'
\ No newline at end of file
+AWS_REGION = 'AWS_REGION'
+THUNDRA_LAMBDA_PUBLISH_REST_BAESURL = 'thundra_lambda_publish_rest_baseUrl'
```

Comment:
THUNDRA_LAMBDA_PUBLISH_REST_BAESURL

should this be 

THUNDRA_LAMBDA_PUBLISH_REST_BASE_URL?